src/mavsdk/core/CMakeLists.txt

```diff
@@ -12,6 +12,29 @@ get_target_property(
 configure_file(version.h.in version.h)
 configure_file(include/mavsdk/mavlink_include.h.in include/mavsdk/mavlink_include.h)
 
+# Generate embedded XML header for libmav
+set(MAVLINK_MINIMAL_XML "${DEPS_INSTALL_PATH}/include/mavlink/message_definitions/v1.0/minimal.xml")
+set(MAVLINK_STANDARD_XML "${DEPS_INSTALL_PATH}/include/mavlink/message_definitions/v1.0/standard.xml")
+set(MAVLINK_COMMON_XML "${DEPS_INSTALL_PATH}/include/mavlink/message_definitions/v1.0/common.xml")
+set(MAVLINK_ARDUPILOTMEGA_XML "${DEPS_INSTALL_PATH}/include/mavlink/message_definitions/v1.0/ardupilotmega.xml")
```

Comment:
I think this creates a hard dependency on the superbuild. Maybe it would be worth having the path as an option, e.g. `MAVLINK_MSG_DEFINITIONS_PATH`, and the superbuild would by default set it to `${DEPS_INSTALL_PATH}/include/mavlink` or something?

Reply:
I thought that too but then it worked in CI...